LOCALIZE.md

```diff
@@ -42,7 +42,7 @@ Each `letter` has a unique id and an optional set of `tags`. Possible values for
 ```
 
 ### 2. Phonemes, syllables and words
-The file `assets/oc-literacy-gen/local/LOCALE/wordcomponents.xml` defines the key phonemes, consonant clusters and syllables present in the language. It also contains a curated set of high-frequency and culturally specfic words for learning the language.
+The file `assets/oc-literacy-gen/local/LOCALE/wordcomponents.xml` defines the key phonemes, consonant clusters and syllables present in the language. It also contains a curated set of high-frequency and culturally specific words for learning the language.
 
 Each `phoneme` has a unique `id` with the prefix `is`. In Swahili, the phonemes _a, th_ and _ng'_ are represented like this:
```

Comment:
Update to use English examples throughout

Reply:
Updated